strings/names/lizard_female.txt

```diff
@@ -2,67 +2,36 @@ Adzi
 Ah
 Ahaht
 Ajim
-Akeenus
-Akish
-Akishan
-Aleeto
 Am
-Amussa
 An
 Anozz
-Asheemar
```

Comment:
ok, literally everything about the lizards name is just an elder scrolls reference, no need to delete all the references here

Reply:
Keep in mind these aren't single word names.